src/features/governance/hooks/useVote.ts

```diff
@@ -73,13 +87,12 @@ export const useVote = () => {
       if (proposal.stage !== ProposalStage.Referendum) {
         throw new Error('vote called on proposal that is not in Referendum stage');
       }
-      if (!voteWeight || voteWeight - MAX_AMOUNT_THRESHOLD <= 0) {
-        showErrorToast(
-          'Insufficient balance when taking in account gas costs. Do you have unlockable stCelo?'
```

Comment:
removing because gas isnt paid in st celo right?

Reply:
correct